docs/podman-create.1.md

```diff
@@ -253,7 +253,7 @@ You need to specify multi option commands in the form of a json string.
 
 Set environment variables
 
-This option allows you to specify arbitrary environment variables that are available for the process that will be launched inside of the container. If you specify a environment variable without a value, podman will check the host environment for a value or set the environment to "". If you specify a environment variable ending in --*--, podman will search the host environment for variables starting with the prefix and add them to the container.  If you want to add an environment variable with a ***** following it, then you need to set a value.
+This option allows arbitrary environment variables that are available for the process to be launched inside of the container.  If an environment variable is specified without a value, Podman will check the host environment for a value and set the variable only if it is set on the host.  If an environment variable ending in __*__ is specified, Podman will search the host environment for variables starting with the prefix and will add those variable to the container.  If an environment variable with a trailing ***** is specified, then a value must be supplied.
```

Comment:
gack, my bad, I'd a typo.   Next to last sentence,  s/add those variable to/add those variables to/   (variable to variables and sorry!)

Reply:
one more.  `trailing ****` is showing four asterisks, should it be showing only one?  Maybe it should be 'trailing __*__`?

Reply:
Nope, forget the 4 astrisks questions, I see now.

Reply:
I don't know. I really confusing about this **** describe. The code can support a variable ends with one or more *